pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -463,6 +464,14 @@ func (p *staticPolicy) allocateCPUs(s state.State, numCPUs int, numaAffinity bit
 
 func (p *staticPolicy) guaranteedCPUs(pod *v1.Pod, container *v1.Container) int {
 	qos := v1qos.GetPodQOS(pod)
+
+	// The CPU manager static policy does not support pod-level resources.
+	if utilfeature.DefaultFeatureGate.Enabled(features.PodLevelResources) && resourcehelper.IsPodLevelResourcesSet(pod) {
```

Comment:
this needs to be checked after QoS check, otherwise the message would be logged for pods with burstable and best-effort QoS classes as well

Reply:
Done, thank you.

Reply:
Please use the same helper `p.isPodWithPodLevelResources(pod)` also here

Reply:
Resolved in https://github.com/kubernetes/kubernetes/pull/133256